vignettes/sentimentanalyzerR.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+title: "sentimentanalyzerR"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{sentimentanalyzerR}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Introduction
+
+This document gives you an overview of the **sentimentanalyzerR** package and its corresponding functions. The package is written to enable beginner level users of R to conveniently translate a given text data into graphical illustrations to draw immediate insights. This is useful especially when the survey is obtained from a large number of respondents spanning across thousands of rows. This package summarizes the major sentiment of the reviews through aggregated scores and plots.
+
+To start using the package, follow the installation section of the README file and install the package. Then import the package using the following code.
+
+```{r load_library}
+library(sentimentanalyzerR)
+```
+
+## Definitions
+
+Compound Score - A score in the range of -1 to 1 that represents the general opinion of a document. It is computed using the valence score for each word in the document. Valence score is a score calculated for each word based on the connotation and context of its usage in a sentence.
+
+Likert Scale - Likert scale is used in questionnaires to enable the respondents to provide a level of agreement or disagreement on a symmetric scale. The scale can be orientated either in the increasing or decreasing order. It has found wide applications in the field of statistics, psychology, business and sciences. Each level of the scale is perceived as evenly spaced in the intensity of agreement/disagreement.
+
```

Comment:
Do we also need to add some detail about `sentiment_score_plot` here?

Reply:
I was defining only the terms that are somewhat technical. I describe the functions in their individual sections below. I was wondering, sentiment score plot isn't a term by itself to define. I can include it if it still feels necessary when I get to it tonight, pls let me know